src/types.ts

```diff
@@ -1,31 +1,32 @@
 /**
  * Get type name of provided variable.
  *
+ * @param {any} value any variable
  * @returns {string} variable type name
- * @param {any} obj any variable
  */
-export function toType(obj) {
+export function toType(value: any): string {
+    // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
```

Comment:
Isn't that rule disabled already by RD-3222?

Reply:
Good point!
Was working on the older branch and didin't thought about that
Thanks for pointing that out!